tests/spread/general/sources/snaps/git-submodules/snapcraft.yaml

```diff
@@ -0,0 +1,15 @@
+name: git-recurse-submodules
+base: core20
```

Comment:
if this general suite runs on core20 this is fine, my brain doesn't feel comfortable with cloning core18 though

Reply:
Agreed.